
# HG changeset patch
# User hans@rhel5-devel.localdomain
# Date 1259226238 -3600
# Node ID a0dee57ef28a5f37ba0a33c98a0cb88de4f9c175
# Parent  3b9e1bf8c8b8ef63517fbd79d4a3689bf2632c72
libv4l: add missing <stdint.h> include to libv4l1.h

From: Hans de Goede <hdegoede@redhat.com>

libv4l: add missing <stdint.h> include to libv4l1.h

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/include/libv4l1.h	Tue Nov 24 09:17:43 2009 +0100
+++ b/v4l2-apps/libv4l/include/libv4l1.h	Thu Nov 26 10:03:58 2009 +0100
@@ -24,6 +24,7 @@
 #endif /* __cplusplus */
 
 #include <stdio.h>
+#include <stdint.h>
 #include <unistd.h>
 
 #if __GNUC__ >= 4

